shell/content/docs/en-us/whats-new.md

```diff
@@ -2,6 +2,33 @@
 title: What's New in 2.6
 ---
 
+
+## New in 2.6.7
+
+### Amazon Billing in the Marketplace
+
+- Now you can purchase support for Rancher through the AWS marketplace 
+
+### Ability to configure Azure AD with MSAL
+
+- [need description]
+
+### Kubernetes 1.24
+
+- Kubernetes 1.24 is now available as a supported Kubernetes version option when provisioning RKE1 clusters as well as upgrading imported RKE2/k3s clusters.
+
+### RKE2 Encryption Key Rotation
+
+- [need description]
+
+
+## New in 2.6.6
+
+### Bug fix for a performance problem
+
+- Addresses a traffic spike from downstream clusters
```

Comment:
Is this for 4484 ? That was released in 2.6.6, but obviously this modal wasn't updates, so is this aiming to cover that ?

Reply:
Yeah, I want to primarily account for why there's a 2.6.6 in this area since we'd skip from 2.6.5 to 2.6.7 without any explanation.

Reply:
Done in another conversation.